less/home-page.less

```
@@ -1,35 +1,76 @@
 .home {
 
+  header {
+    border-bottom: 2px dashed @silver;
+
+    @media(max-width:800px){
+
+ }
+
+
+
+  .intro {
+    border-bottom: 1px dashed @shark;
+    
+    @media(max-width:500px) {
```

Would have liked to have seen interpolation.
```suggestion
    @media @phone{
```